docs/charts/index.yaml

```diff
@@ -1,46 +1,46 @@
 apiVersion: v1
```

Comment:
I didn't change this. Is there an easy way to remove it from the commit, or should I just ignore it? (Ditto the tgz)